splittable.js

```diff
@@ -238,9 +238,10 @@ exports.getGraph = function(entryModules, config) {
   // directly and which we don't want to apply during deps finding.
   transform(babel, {
     babelrc: false,
-    plugins: [
+    plugins: Array.isArray(config.babel.plugins) ? config.babel.plugins.concat([
```

Comment:
Should this be a new config option or the same as above?

Reply:
Lets rename then.

Reply:
should we allow for 2 babel configs or should a hacky `config.babel.depSearchPhasePlugins` do?

Reply:
Can we maybe make the plugin you need default?

Reply:
ideally it shouldn't since the "use strict" directive is desirable after all (correct me if im wrong), its just unfortunate that jison uses a lot of labels and jumps